src/Models/ElementContent.php

```diff
@@ -29,9 +31,20 @@ class ElementContent extends BaseElement
     public function getCMSFields()
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $fields
-                ->fieldByName('Root.Main.HTML')
-                ->setTitle(_t(__CLASS__ . '.ContentLabel', 'Content'));
+            // Configure a slimmed down HTML editor for use with blocks
+            /** @var HTMLEditorField $editorField */
+            $editorField = $fields->fieldByName('Root.Main.HTML');
+            $editorField->setRows(7);
```

Comment:
The reason it was in the original place is because it then only affects inline editing. This would also affect non inline editable elements, which naturally have a whole window to fill so a default size would be acceptable

Reply:
Ah yep of course. I even did consider adding a `if ($this->inlineEditable())` statement in there! Ok I'll restore this back to it's original spot and I might as well move the status-bar removal there.

Reply:
Adjusted 👍

Reply:
🙏